drf_kit/tests.py

```diff
@@ -117,6 +117,14 @@ def assertResponseNotAllowed(self, response):
             response_key=None,
         )
 
+    def assertResponseAccepted(self, response):
+        self.assertResponse(
+            expected_status=status.HTTP_202_ACCEPTED,
+            expected_body="",
```

Comment:
This should be parameterized, it's common to return 202 with some ID for further tracking.
It can be `Any` by default to handle simpler use cases.